packages/graphql-codegen-cli/src/load.ts

```diff
@@ -52,22 +52,17 @@ export async function loadSchema(
     return schema;
   } catch (e) {
     throw new Error(
-      `
-        Failed to load schema from ${Object.keys(schemaPointers).join(',')}:
-
-        ${e.message || e}
-        ${e.stack || ''}
-
-        GraphQL Code Generator supports:
-          - ES Modules and CommonJS exports (export as default or named export "schema")
-          - Introspection JSON File
-          - URL of GraphQL endpoint
-          - Multiple files with type definitions (glob expression)
-          - String in config file
-
-        Try to use one of above options and run codegen again.
-
-      `
+      [
+        `Failed to load schema from ${Object.keys(schemaPointers).join(',')}:`,
+        printError(e),
+        '\nGraphQL Code Generator supports:',
+        '\n- ES Modules and CommonJS exports (export as default or named export "schema")',
+        '- Introspection JSON File',
+        '- URL of GraphQL endpoint',
+        '- Multiple files with type definitions (glob expression)',
+        '- String in config file',
+        '\nTry to use one of above options and run codegen again.\n',
+      ].join('\n')
```

Comment:
I find joining strings like this is easier to read and formatted better in the terminal

Reply:
YES!